docs/migration/4_0.md

```diff
@@ -1207,6 +1208,15 @@ related labels outside of checkout and the checkout lib is not used, you will ne
 `RoutingService.go` - Removed 2nd argument `query`. Use `extras.queryParams` instead.
 `RoutingService.navigate` - Removed 2nd argument `query`. Use `extras.queryParams` instead.
 
+### RoutingActions ngrx
+The following ngrx actions has been deprecated:
+- `RoutingActions.RouteGo` (and `RoutingActions.ROUTER_GO`)
+- `RoutingActions.RouteGoByUrlAction` (and `RoutingActions.ROUTER_GO_BY_URL`)
+- `RoutingActions.RouteBackAction` (and `RoutingActions.ROUTER_BACK`)
+- `RoutingActions.RouteForwardAction` (and `RoutingActions.ROUTER_FORWARD`).
+
+Please just use the methods of the `RoutingService`, respectively: `go()`, `goByUrl()`, `back()`, `forward()`.
+
```

Comment:
In my opinion it's worth to mention here about changes in those methods. For example that they are returning promise now instead of boolean, additional params etc.

Reply:
In the migration doc, I'd focus on explaining breaking changes to customers and giving hints how to fix them.

- Changing the return type of methods from void to Promise, is not breaking. It's a new feature btw.
- Only optional parameters were added to one method - so it's also not breaking.